docs/developer-docs/integrations/sns/launching/index.md

```diff
@@ -1,18 +1,13 @@
 # Launching an SNS
 
 This Section describes the SNS launch process in detail.
-It starts with a detailed description of all [Stages of an SNS launch](./launch-summary.md).
-Then, it explains the practical [commands & actions to go through SNS launch](./launch-steps.md), in terms of concrete tools and commands that are needed to take a project through all of these stages and launch an SNS for it.
-
-<!-- NEW: 
 It starts with a detailed description of all Stages of an SNS launch.
 Then, it explains the practical commands & actions to go through SNS launch, in terms of concrete tools and commands that are needed to take a project through all of these stages and launch an SNS for it.
 
-On August 14th, 2023, the NNS has decided to release a new version of the SNS with a new, simplified launch process. In particular, the new launch process only requires one single NNS proposal that will, if adopted, trigger the creation of the SNS canisters as well as starting and finishing the decentralization swap. Since the new flow is considerably simpler both for developers who want to hand over their dapp to the IC and for voters/users, it is recommended to use the new flow.
+In August 2023, the NNS has decided to release a new version of the SNS with a new, simplified launch process. In particular, the new launch process only requires one single NNS proposal that will, if adopted, trigger the creation of the SNS canisters as well as starting and finishing the decentralization swap. Since the new flow is considerably simpler both for developers who want to hand over their dapp to the IC and for voters/users, it is recommended to use the new flow.
```

Comment:
```suggestion
In August 2023, the NNS voted to release a new version of the SNS with a new, simplified launch process. In particular, the new launch process only requires one single NNS proposal that will, if adopted, trigger the creation of the SNS canisters as well as starting and finishing the decentralization swap. Since the new flow is considerably simpler both for developers who want to hand over their dapp to the IC and for voters/users, it is recommended to use the new flow.
```